Dev

MsQCompSci opened the pull request.

Commit:
INIT COMMIT

sbellows1 committed:
Visualizations and slight cleans for df_basic

sbellows1 committed:
More visualizations, function for plotting side dfs

sbellows1 committed:
side df function working

sbellows1 committed:
maybe the function will go this time??

sbellows1 committed:
function applied across all side dfs

Shampjeff committed:
updates to kaggle survey

Shampjeff committed:
Revert "updates to kaggle survey"

This reverts commit 500b84e36642857401832a238df44452a2ded1bc.

Shampjeff committed:
Update README.md

Shampjeff committed:
add layla scraping file

Shampjeff committed:
updates to files

sbellows1 committed:
Merge branch 'dev' of https://github.com/Shampjeff/data_607_project3

Shampjeff committed:
Merge pull request #5 from sbellows1/dev

Cleans and visualizations

sbellows1 committed:
automatically selects features that increase salary.

sbellows1 committed:
deleting dumb list

Shampjeff committed:
Merge branch 'dev' into dev

Shampjeff committed:
Merge pull request #7 from sbellows1/dev

Automated selection of graphs that improve median salary

Shampjeff committed:
kmeans on kaggle data

MsQCompSci committed:
Rmd to dev

Shampjeff committed:
visuals for clusters

Shampjeff committed:
Merge branch 'dev' of https://github.com/Shampjeff/data_607_project3 into dev
Merge

Shampjeff committed:
kmeans update and some narrative

sbellows1 committed:
Visualizations cleaned and presentable. Last thing will be final storytelling/results aggregation.

sbellows1 committed:
Merge branch 'dev' of https://github.com/Shampjeff/data_607_project3

Commit:
Kaggle Backend finalized

Commit:
next commit

Shampjeff committed:
Merge pull request #9 from sbellows1/dev

visualizations done

Shampjeff commented:
I'm not sure this PR is going to do what we want. This will move all of the last 23 commits from me, you, and Sam to the master branch. 

I would do what you did last time and open a PR for the master branch and put the updated files there. We can either move them later or just keep them there.

Shampjeff committed:
Merge branch 'dev' into master

Shampjeff committed:
Merge pull request #10 from AngelClaudio/master

backend work final

Shampjeff committed:
updates for kmeans

sbellows1 committed:
Merge branch 'dev' of https://github.com/Shampjeff/data_607_project3

Commit:
final backend commit

Commit:
commit

Merge branch 'master' of https://github.com/AngelClaudio/data_607_project3

# Conflicts:
#	kaggle_survey_clean_write.Rmd

Commit:
Delete kaggle_survey_clean_write.Rmd

Commit:
Delete poc_kaggle_data.R

Commit:
Delete poc_proj_three.Rmd

Shampjeff committed:
initial kmeans final, sql uupload kmeans backend

MsQCompSci committed:
Consolidated Scraping Rough Draft

Shampjeff committed:
conflictt res hopefully

Shampjeff committed:
Merge branch 'AngelClaudio-master' into dev

sbellows1 committed:
This is a final version unless any issues come up

sbellows1 committed:
Delete KagglePartTwo.Rmd

sbellows1 committed:
Final version unless issues

Shampjeff committed:
finish work on kmeans

Shampjeff committed:
Merge branch 'dev' into dev

Shampjeff committed:
Merge pull request #12 from sbellows1/dev

Final unless issues arise

sbellows1 committed:
final file

Shampjeff committed:
Merge pull request #13 from sbellows1/dev

final file

Shampjeff committed:
update to kmeans
